lib/core/src/Cardano/Pool/DB.hs

```diff
@@ -109,13 +112,26 @@ data DBLayer m = forall stm. (MonadFail stm, MonadIO stm) => DBLayer
         -> stm (Maybe PoolRegistrationCertificate)
         -- ^ Find a registration certificate associated to a given pool
 
+    , nextPoolMetadata
+        :: stm [(PoolId, StakePoolMetadataUrl, StakePoolMetadataHash)]
```

Comment:
`unfetchedPoolMetadata`?

is there a `LIMIT`?

Reply:
There's no limit, this potentially returns 1000 items at the moment. 

> unfetchedPoolMetadata

:+1: thanks

Reply:
Could perhaps even do `unfetchedPoolMetadataReferences`

Reply:
`unfetchedPoolMetadataRefs`